macrosynergy/download/jpmaqs.py

```diff
@@ -486,7 +487,14 @@ def download(
                 "mop_lag",
                 "grading",
             ], f"Incorrect metric passed: {metric}."
+            
+        assert isinstance(start_date, str), "Start date must be a string in the format YYYY-MM-DD"
+        if end_date is not None:
+            assert isinstance(end_date, str), "End date must be a string in the format YYYY-MM-DD"
+        else:
+            end_date = (datetime.datetime.today() + pd.offsets.BusinessDay(2)).strftime("%Y-%m-%d")
```

Comment:
Ah, this won't work Palash, unfortunately DQ doesn't take a ISO format date, but rather "%y%m%d" as the string - if you check the docs.